cmd/distributedfetch/distributedfetch.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"log"
+
+	sf "github.com/snowflakedb/gosnowflake"
+)
+
+func main() {
+	defer printSummary()
+	if !flag.Parsed() {
+		flag.Parse()
+	}
+
+	cfg, err := sf.GetConfigFromEnv([]*sf.ConfigParam{
+		{Name: "Account", EnvName: "SNOWFLAKE_TEST_ACCOUNT", FailOnMissing: true},
+		{Name: "User", EnvName: "SNOWFLAKE_TEST_USER", FailOnMissing: true},
+		{Name: "Password", EnvName: "SNOWFLAKE_TEST_PASSWORD", FailOnMissing: true},
+		{Name: "Host", EnvName: "SNOWFLAKE_TEST_HOST", FailOnMissing: false},
+		{Name: "Port", EnvName: "SNOWFLAKE_TEST_PORT", FailOnMissing: false},
+		{Name: "Protocol", EnvName: "SNOWFLAKE_TEST_PROTOCOL", FailOnMissing: false},
+	})
+	if err != nil {
+		log.Fatalf("failed to create Config, err: %v", err)
+	}
+	dsn, err := sf.DSN(cfg)
+	if err != nil {
+		log.Fatalf("failed to create DSN from Config: %v, err: %v", cfg, err)
+	}
+
+	db, err := sql.Open("snowflake", dsn)
+	if err != nil {
+		log.Fatalf("failed to connect. %v, err: %v", dsn, err)
+	}
+	defer db.Close()
+
+	sf.GetLogger().SetLogLevel("debug")
+
+	query := "SELECT 1 FROM TABLE(GENERATOR(ROWCOUNT=>10000000))"
+	rows, err := db.Query(query) // no cancel is allowed
+	if err != nil {
+		log.Fatalf("failed to run a query. %v, err: %v", query, err)
+	}
+	defer rows.Close()
+	var v int
+	for rows.Next() {
+		err := rows.Scan(&v)
+		if err != nil {
+			log.Fatalf("failed to get result. err: %v", err)
+		}
+		if v != 1 {
+			log.Fatalf("failed to get 1. got: %v", v)
+		}
+	}
+	if rows.Err() != nil {
+		fmt.Printf("ERROR: %v\n", rows.Err())
+		return
+	}
+
+	sf.GetLogger().SetLogLevel("error")
+}
+
+func printSummary() {
+	fmt.Printf(`
+		Please observe as follows.
```

Comment:
This sentence is obsolete. Additionally I would inform a reader about what are the configuration possibilities here (batches size etc.). Also i think this is more of a exampleDescription and not a summary.

Reply:
1. Removed first line.
2. Renamed function.
3. I don't think that we should add configuration possibilities. We haven't done that in other examples. Also, this is summary, not configuration instructions :)

Reply:
Agreed, LGTM